File: read_lines/read.c
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>

int main(void){

    ssize_t read;
    size_t len = 0;

    char *line;
    FILE *fp = fopen("hello.txt", "r");
    while(1){
	    while ((read = getline(&line, &len, fp)) != -1) {
	        printf("%s", line);
	        sleep(1);
	    }
	    fseek(fp, SEEK_SET, 0);
	}
    fclose(fp);

    return 0;
}

File: read_lines/Read.java
import java.io.BufferedReader;
import java.io.FileReader;
import java.io.IOException;


public class Read {

	public static void main(String [] args){
		
		try {
                    while(true){
			BufferedReader fileReader = new BufferedReader(new FileReader("hello.txt"));
			for(String s = fileReader.readLine(); s != null; s = fileReader.readLine()){
				System.out.println(s);
                                Thread.sleep(1000);
                        }
			fileReader.close();
                    }
		} catch (Exception e) {
			e.printStackTrace();
		}
		
	}
}
